ios/RNSPeekableView.m

```diff
@@ -0,0 +1,166 @@
+#import "RNSPeekableView.h"
+#import "RNSScreenContainer.h"
+#import <React/RCTUIManagerObserverCoordinator.h>
+#import <React/RCTUIManager.h>
+#import "RNSScreen.h"
+#import <React/RCTUIManagerUtils.h>
+
+
+@implementation RNPreviewViewController {
+  RNSPeekableWrapper *_wrapper;
+}
+
+
+- (instancetype)initWithWrapper:(RNSPeekableWrapper *) wrapper
+{
+  if (self = [super init]) {
+    _wrapper = wrapper;
+  }
+  return self;
+}
+
+- (UIViewController *)previewingContext:(id<UIViewControllerPreviewing>)previewingContext viewControllerForLocation:(CGPoint)location
+{
+  if (_wrapper.onPeek) {
+    _wrapper.onPeek(@{});
```

Comment:
can't we just do `_wrapper.onPeek(nil)` instead of this?

Reply:
nah, it's needed